24-Content Placeholder/script.js

```diff
@@ -0,0 +1,27 @@
+const header = document.getElementById('header')
+const title = document.getElementById('title')
+const excerpt = document.getElementById('excerpt')
+const profile_img = document.getElementById('profile_img')
+const namee = document.getElementById('name')
+const date = document.getElementById('date')
+
+const animated_bgs = document.querySelectorAll('.animated-bg')
+const animated_bg_text = document.querySelectorAll('.animated-bg-text')
+
+setTimeout(getData, 2500)
+
+function getData() {
+  header.innerHTML = '<img src="https://images.unsplash.com/photo-1539376248633-cf94fa8b7bd8?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80"alt="Laptop image"/>'
+
+  title.innerHTML = 'Lorem ipsum dolor sit amet'
+
+  excerpt.innerHTML = 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos, quo'
+
+  profile_img.innerHTML = ' <img src="https://randomuser.me/api/portraits/men/45.jpg" alt="profile-img" />'
+
+  namee.innerHTML = 'John Doe'
+  date.innerHTML = 'Dec 04 , 2022'
+
+  animated_bgs.forEach(bg => bg.classList.remove('animated-bg'))
+  animated_bg_text.forEach(bg => bg.classList.remove('animated-bg-text'))
+}
```

Comment:
I have a feeling like you were rushing through this? I think you can do better.
for one, let's pretend that there is a function called `getData()` that will return data object.
here is is:
```
const data = {
  headerImage: {
    name: 'Laptop image',
    url: 'https://images.unsplash.com/photo-1539376248633-cf94fa8b7bd8?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80'
  },
  title: 'Lorem ipsum dolor sit amet',
  body: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos, quo',
  profile: {
    name: '',
    image: {
      name: 'profile-img',
      url: 'https://randomuser.me/api/portraits/men/45.jpg'
    },
    date: {
      day: 4,
      month: 12,
      year: 2022
    }
  }
}
```

now we need to use this data to insert it inside the HTML

Reply:
or even better go ahead and implement an `async/await` functionality and reterive a `user` from `https://jsonplaceholder.typicode.com/`